src/components/chat/chat.js

```diff
@@ -4,11 +4,10 @@ import CancelIcon from '@material-ui/icons/Cancel';
 import MailOutlineIcon from '@material-ui/icons/MailOutline';
 import MessengerCustomerChat from 'react-messenger-customer-chat';
 import { useSelector } from 'react-redux';
-import { Button, TextField } from '@material-ui/core';
+import { FOOTER_CONTACTS } from '../../translations/footer.translations';
+import { ActiveMessenger } from './activeMessenger';
 import { useStyles } from './chat.style';
 import { CHAT } from '../../translations/chat.translation';
-import kyivstar from '../../images/kyivstar.png';
-import vodafone from '../../images/vodafone.png';
 
 export const Chat = () => {
   const [visible, setBlock] = useState(false);
```

Comment:
What does `visible` reflect? As I understand it allows to hide all icons, doesn't it?

Reply:
Yes, for a better understanding of what this variable does, I passed the variable 'visible' to 'iconsVisible'